documentation/modules/con-kafka-bridge-logging.adoc

```diff
@@ -0,0 +1,74 @@
+z// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+[id='con-kafka-bridge-logging-{context}']
+= Kafka Bridge loggers
+
+Kafka Bridge has a configurable logger for each OpenAPI operation, which are formatted as follows:
```

Comment:
Suggested change:

Kafka Bridge has configurable loggers for each OpenAPI operation, which are formatted as follows:

Reply:
This section has changed slightly, as we now nclude another logger is the list.
So I changed to:

Kafka Bridge has its own configurable loggers:

* `log4j.logger.io.strimzi.kafka.bridge`
* `log4j.logger.http.openapi.operation.__<operation-id>__`

The `log4j.logger.http.openapi.operation.__<operation-id>__` logger is for each OpenAPI operation, where `_<operation-id>_` is the identifier of a specific operation.

Here are the operations defined by the OpenAPI specification: